Tests/MapboxDirectionsTests/RouteRefreshTests.swift

```diff
@@ -175,7 +175,7 @@ class RouteRefreshTests: XCTestCase {
     
     func testEncoding() {
         let leg = RefreshedRouteLeg(attributes:
-            .init(segmentDistances: [0], expectedSegmentTravelTimes: [0], segmentSpeeds: [0], segmentCongestionLevels: [.severe], segmentMaximumSpeedLimits: [Measurement(value: 1, unit: .milesPerHour)]))
+            .init(segmentDistances: [0], expectedSegmentTravelTimes: [0], segmentSpeeds: [0], segmentCongestionLevels: [CongestionLevel.severe], segmentMaximumSpeedLimits: [Measurement(value: 1, unit: .milesPerHour)]))
```

Comment:
A new test is failing to build in Xcode 10.2 due to what appears to be a compiler bug:

```
/Users/distiller/project/Tests/MapboxDirectionsTests/RouteRefreshTests.swift:178:14: error: type 'RouteLeg.Attributes' has no member 'severe'
            .init(segmentDistances: [0], expectedSegmentTravelTimes: [0], segmentSpeeds: [0], segmentCongestionLevels: [.severe], segmentMaximumSpeedLimits: [Measurement(value: 1, unit: .milesPerHour)]))
             ^~~~                                                                                                       ~~~~~~~
```

I’ve worked around the issue by adding an explicit type that shouldn’t need to be there.